src/common/replayBrowser/ReplayServer.ts

```diff
@@ -0,0 +1,43 @@
+import { ipcMain } from "electron";
+
+import { getFullReplay, getPlayerReplays, pruneFolders } from "./dao";
```

Comment:
What does `dao` mean? I assume it's not "decentralized autonomous organization". Also on the topic of naming files, only React components should start with a capital letter. e.g. `SomeReactComponent.tsx` versus `someLibraryFunctions.ts`

Reply:
Lol that's my Java showing, it stands for "Data Access Object". db.ts is probably a more suitable name
https://stackoverflow.com/questions/19154202/data-access-object-dao-in-java

Reply:
lovely :D